hamster.js

```diff
@@ -2,24 +2,28 @@ var request = require('request');
 var open = require('open');
 
 // 1. Find the correct URL to get adorable hamsters from Giphy!
-request('GIPHY API GET REQUEST URL', function (error, response, body) {
+request('http://api.giphy.com/v1/gifs/random?api_key=dc6zaTOxFJmzC&q=hamsters', function (error, response, body) {
   if (!error && response.statusCode == 200) {
     var response_hash = JSON.parse(body);
     console.log(response_hash);
 
+
     // 2. Set up an array variable `data_array` that stores the array of GIF hashes. HINT: You'll want to grab the value of the "data" key in the `response_hash` object.
-    // var data_array = ???;
+
+var data_array = []
+data_array.push(response_hash)
 
     // 3. Use the `map()` function to iterate through the `data_array` array. For each `gif_hash` in `data_array`, return the GIF URL.
-    // var url_array = data_array.map(function(gif_hash){
-    //   ???
-    // })
+
+var url_array = data_array.map(function(gif_hash){
+  return 'http://api.giphy.com/v1/gifs/search?q=' + gif_hash + '&api_key=dc6zaTOxFJmzC'
```

Comment:
This won't work:

`git_hash` is an object here, so you'll need to pull a url property off that object and return it.